modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/ItDistributedConfigurationPropertiesTest.java

```diff
@@ -286,6 +287,8 @@ void setUp(@WorkDirectory Path workDir, TestInfo testInfo) throws Exception {
         Stream.of(firstNode, secondNode).parallel().forEach(Node::start);
 
         firstNode.cmgManager.initCluster(List.of(firstNode.name()), List.of(), "cluster");
+
+        Stream.of(firstNode, secondNode).parallel().forEach(Node::deployWatches);
```

Comment:
Same as above. It's required to call deployWatches within node's startup process and assert deployWatches' future completion after init.